xml/art-obs-beginners-guide.xml

```diff
@@ -175,24 +167,28 @@ https://www.suse.com/communities/blog/suse-studio-integration/
    <title>Projects and Packages</title>
    <para>
     In &obsa;, packages are organized in <emphasis>projects</emphasis>.
-    A single project can contain several packages and serves a specific
-    purpose, defines access control, other repositories, and build targets
-    (operating systems and architectures).
+    A single project can contain several packages and usually serves a specific
+    organizational purpose. Generic access control, related repositories, and
+    build targets (operating systems and architectures) are all defined at
+    the project level.
    </para>
    <para>
     Projects can also contain other projects (subprojects) to structure work.
     Each project name is separated by colons.
```

Comment:
That mean, we have to introduce "maintainers" as a new term. This is one of the terms which I was unsure how to define it (still left in the glossary as "to be defined" 😉).
Do you have a suggestion?

If you think this is important and maintainers has to be mentioned, then I would like to add something about maintainers in the "Projects and Packages" section.